app.py

```diff
@@ -0,0 +1,61 @@
+import json
+from functools import wraps
+
+from flask import Flask, jsonify, abort, request
+
+def create_app():
+
+    settings = {
+        'DEBUG': True,
+    }
+
+    app = Flask(__name__)
+    app.config.update(settings)
+
+    return app
+
+app = create_app()
+
+
+def get_books():
+    books = {}
+
+    return books
+
+
+def save_book(book):
```

Comment:
I don't think this function is actually doing anything. You might want to add append it to a list.